servicex/query_core.py

```diff
@@ -551,21 +557,27 @@ async def get_signed_url(
             if progress:
                 progress.advance(task_id=download_progress, task_type="Download")
 
+        later_than: datetime.datetime | None = None
```

Comment:
Why not initialize `later_than` to `datetime.min` to avoid all of the extra `if later_than is None or` constructs?